src/tab.group.ts

```diff
@@ -58,18 +58,17 @@ export default class GlideCoreTabGroup extends LitElement {
           this.isShowOverflowButtons,
           () => html`
             <button
-              style="height: ${this.#tabListElementRef.value?.clientHeight}px"
```

Comment:
Any idea why this would be needed? It doesn't seem to do anything.

Reply:
Yeah that's interesting.  I think removing this is fine!